utils/cams_wind_pipeline.py

```
@@ -0,0 +1,265 @@
+        """Save xarray Dataset to PostgreSQL database, drop and recreate table, insert in chunks, and delete files."""
+        try:
+            df = ds.to_dataframe().reset_index()
+            df.columns = [col.lower() for col in df.columns]
+
+            if df['time'].dtype == 'object' and any(isinstance(t, cftime.DatetimeGregorian) for t in df['time']):
+                df['time'] = pd.to_datetime([t.isoformat() for t in df['time']])
+
+            expected_columns = {'time', 'latitude', 'longitude', 'wind_speed', 'wind_direction'}
+            if not expected_columns.issubset(df.columns):
+                logger.error(f"DataFrame missing required columns: {expected_columns - set(df.columns)}")
+                raise ValueError(f"DataFrame missing required columns")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)